index.php

```diff
@@ -3,6 +3,11 @@
 
 require_once( 'vendor/autoload.php' );
 
+use Dotenv\Dotenv as Dotenv;
```

Comment:
I came to this syntax by trial and error. I think it's correct because `index.php` runs and the linter doesn't bark at me. I don't know if it's the best way to write this, though.

I used the `as` just to avoid writing `Dotven\Dotenv` again.

Reply:
Odd that you'd need to use the `as` 🤔 

I have a suggestion I'll leave in the next comment :)

Reply:
Uhm yeah, actually, I don't seem to need the `as`.

As I said, I was doing some trial and error, so I must have missed something along the way.

Thanks 😄